client/src/components/Trip/AddTrip.tsx

```diff
@@ -154,41 +154,42 @@ const AddTrip = () => {
       <form onSubmit={handleStartTrip} className=''>
         <div>
           {visibleDiv == 'trip' ? (
-            <div onClick={() => changeVisibleDiv('')}>
-              <div className='ToggleDiv w-[95%] h-[150px] mt-4 bg-stone-50 rounded-[20px] shadow-lg border-voyagrBorders border p-2 flex mx-auto mb-5'>
-                <label className='w-full text-zinc-800 text-3xl font-normal font-noto'>
+            <div 
+            // onClick={() => changeVisibleDiv('')} //wtf?

```

Comment:
This is a place of work. Please refrain from using comments with this type of language. Voyageurs should know better. I will have to report this to your manager.